device_calendar/android/src/main/kotlin/com/builttoroam/devicecalendar/models/RecurrenceRule.kt

```diff
@@ -0,0 +1,10 @@
+package com.builttoroam.devicecalendar.models
+
+import com.builttoroam.devicecalendar.common.RecurrenceFrequency
+
+
+class RecurrenceRule(val recurrenceFrequency : RecurrenceFrequency) {
+    var totalOccurrences: Int? = null
+    var interval: Int? = null
```

Comment:
I think this needs more revision as I don't feel this is advanced enough to handle the variety of recurrence patterns that need to be supported.

Reply:
For the time being we'll merge this PR in order to provide initial support for recurring events.